kodilla-sudoku/src/main/java/com/kodilla/sudoku/board/creator/BoardCreator.java

```diff
@@ -12,7 +12,7 @@ public BoardCreator(SudokuBoard board) {
 
     public void createBoard(int boardSize, int blockSize) {
```

Comment:
przedziwne ... jak coś ma zrobić create, to musi to co stworzyło zwrócić. Wpp. nie jest kreatorem :/

Reply:
Dlaczego w konstruktorze kreator dostaje coś, co ma stworzyć?